addons/portal/__manifest__.py

```diff
@@ -42,7 +42,7 @@
             'portal/static/src/js/**/*',
             'portal/static/src/xml/**/*',
             'portal/static/src/signature_form/**/*',
-            'portal/static/src/chatter/boot/boot_service.js',
+            'portal/static/src/chatter/portal/boot_service.js',
```

Comment:
Maybe boot_service should be top level as we don't need to load it again in lazy portal bundle?